cli/app.go

```diff
@@ -48,6 +48,8 @@ const (
 	moduleFlagPlatform        = "platform"
 	moduleFlagForce           = "force"
 	moduleFlagBinary          = "binary"
+	moduleFlagLocal           = "local"
```

Comment:
Curious: what's the use-case for `--local`? If the RDK is running on the same machine as the RDK, is the point of `--local` to just hotswap the module while your RDK is running (using the same entrypoint)?

Reply:
for python specifically, 'run in place' can be a lot cheaper than 'bundle and install' (because you incur either pyinstaller or venv cost)

even for golang there's a bit of savings from not tarring a large binary